api/v1alpha1/flinkcluster_types.go

```diff
@@ -110,6 +110,21 @@ type JobManagerPorts struct {
 	UI *int32 `json:"ui,omitempty"`
 }
 
+// JobManagerIngress defines ingress of JobManager
+type JobManagerIngressSpec struct {
+	// Ingress host format. ex) {{$clusterName}}.example.com
+	IngressHostFormat string `json:"ingressHostFormat"`
```

Comment:
"Ingress" is redundant in this context, just use "HostFormat".

Reply:
Fixed.